ros/angel_system_nodes/angel_system_nodes/audio/asr.py

```diff
@@ -4,12 +4,13 @@
 import requests
 import soundfile
 import tempfile
+from termcolor import colored
```

Comment:
I see that the `termcolor` package is incidentally in our container environment because of our current dependency on `detectron2`. However, since we are now using this directly, this should become a main dependency of our project. If our `pyproject.toml` file, we should add `termcolor = "^2"` to the `[tool.poetry.dependencies]` section. Then, while in the `./angel-workspace-shell.sh` environment, `poetry lock --no-update` should be run to appropriately update the `poetry.lock` file.

I can also just make this change and push it to your branch here if desired.